tournamentcontrol/competition/tests/test_competition_admin.py

```diff
@@ -1560,3 +1555,75 @@ def test_edit_match_youtube_api_basic_guard_clause(self):
             match.refresh_from_db()
             self.assertFalse(match.live_stream)
             self.assertIsNone(match.external_identifier)
+
+    @mock.patch("tournamentcontrol.competition.models.Season.youtube")
+    def test_edit_match_youtube_refresh_error_handling(self, mock_youtube):
+        """
+        Test that RefreshError from expired OAuth2 tokens is properly handled.
+
+        When a YouTube OAuth2 token expires, the API call raises RefreshError.
+        This should be caught and result in a user-friendly error message
+        rather than crashing the view.
+
+        Refs: https://github.com/goodtune/vitriolic/issues/147
+        """
+        from google.auth.exceptions import RefreshError
```

Comment:
Imports should never be inline, unless they are required to prevent a import cycle.